remote_test.go

```diff
@@ -1633,6 +1634,40 @@ func testSwitchFrame(t *testing.T, c config) {
 	}
 }
 
+func titleChangeCondition(wd WebDriver) (bool, error) {
+	title, err := wd.Title()
+	if err != nil {
+		return false, err
+	}
+
+	fmt.Printf("title \"%s\"\n", title)
+	return title == "Title changed.", nil
+}
+
+func testWait(t *testing.T, c config) {
+	wd := newRemote(t, c)
+	defer quitRemote(t, wd)
+
+	if err := wd.Get(serverURL + "/title"); err != nil {
+		t.Fatalf("wd.Get(%q) returned error: %v", serverURL, err)
+	}
+
+	// Testing when the title should change.
+	if err := wd.Wait(titleChangeCondition); err != nil {
+		t.Fatalf("wd.Wait(titleChangeCondition) returned error: %v", err)
+	}
+
+	// Reloading the page
```

Comment:
Nit: Be consistent with the surrounding code in punctuation in these comments. Use a single period.